awscrt/mqtt.py

```diff
@@ -703,6 +720,30 @@ def puback(packet_id, error_code):
 
         return future, packet_id
 
+    def get_stats(self):
+        """Queries the connection's internal statistics for incomplete operations.
+
+        Returns:
+            A future with a (:class:`OperationStatisticsData`)
```

Comment:
Why is this returning a `future`? A future is good for an async function, but the `aws_mqtt_client_connection_get_stats()` C call seems like it's synchronous. Couldn't we just return a `OperationStatisticsData` directly?

it's not too late to change this, since it JUST got merged

Reply:
PR made to adjust this: https://github.com/awslabs/aws-crt-python/pull/439